workday/12.1/modules/ROOT/pages/workday-connector-troubleshoot.adoc

```diff
@@ -0,0 +1,54 @@
+= Troubleshoot Workday Connector
+
```

Comment:
Are you expecting this topic to gain subtopics over time?
Even if so, consider upleveling the subtopic and subsequent subs:
```suggestion
= Troubleshoot Studio Timeout
```

Reply:
Note that Mysti and I are working on a Troubleshooting topic template that does not match the one currently on our wiki, which is too complicated. We are debating the need for Cause, and for using subtitles at all.

For example, this topic could begin...:
```suggestion
= Troubleshoot Studio Timeout
```
Due to connectivity issues, you might experience a timeout while obtaining ....
... `at
----`

When one of these errors occurs, follow these steps....

With no subheads. Or, using only Issue/Symptoms/Xxxx for what the user sees or experiences (an error, or a stoppage, for example) and Solution/Corrective Action, with steps.

Something to think about.
I'll copyedit what's here, but I think we can make it more focused.

Reply:
I actually expect more troubleshooting sections to be added to this topic so leaving this as H2